backwardcompatibilityml/helpers/training.py

```diff
@@ -313,7 +313,7 @@ def get_error_instance_indices(model, batched_evaluation_data, batched_evaluatio
         return model_diff.nonzero().view(-1).tolist()
 
 
-def get_all_error_instance_indices(h1, h2, batched_evaluation_data, batched_evaluation_target,
+def get_all_error_instance_indices(h1, h2, batch_ids, batched_evaluation_data, batched_evaluation_target,
```

Comment:
We don't need `batch_ids` anymore right? Solets revert this function to its original form.